tigerpath/majors_and_certificates/majors/POL_2018.json

```
@@ -15,197 +15,431 @@
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "The CP sub-field focuses on the similarities and differences in patterns of politics around the world with attention to what happens within states regarding representation, economic development, violence, and effective government.",
+              "course_list": [
+                "POL 230",
+                "POL 338",
+                "POL 349",
+                "POL 351",
+                "POL 352",
+                "POL 355",
+                "POL 360",
+                "POL 366",
+                "POL 367",
+                "POL 374",
+                "POL 384",
+                "POL 432",
+                "POL 433",
+                "POL 434",
+                "POL 479"
+              ]
+            },
+            {
+              "name": "International Relations",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "The IR sub-field focuses on the study of politics among nations and non-state actors in world affairs including subjects such as the causes of war, the role of international law and institutions, economic interdependence, and cooperation to advance common goals for human rights and environmental protection.",
+              "course_list": [
+                "POL 240",
+                "POL 380",
+                "POL 385",
+                "POL 386",
+                "POL 388",
+                "POL 393",
+                "POL 396",
+                "POL 399",
+                "POL 440",
+                "POL 442"
+              ]
+            },
+            {
+              "name": "Methods",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "(Cannot be the primary field). These courses provide undergraduate students with analytical tools they can use to conduct rigorous social science research.",
+              "course_list": [
+                "POL 250",
+                "POL 341",
+                "POL 345",
+                "POL 346",
+                "POL 347"
+              ]
+            }
           ]
         }
       ]
     },
     {
-      "name": "Analytic Requirement",
+      "name": "Departmentals",
       "max_counted": 1,
-      "min_needed": 1,
-      "explanation": "The department maintains a list of politics courses that have an emphasis on methodological tools for research in political science. Concentrators are required to take a course to fulfill the analytic requirement, normally no later than the first term of their junior year. The courses used to fulfill the analytic requirement cannot be used to fulfill primary-, secondary-, or third-field requirements. The analytic requirement may be satisfied by POL 341, POL 345/SOC 305, POL 346, or POL 347. We will also accept ANT 300A, ANT 301A, ECO 202, ECO 302, ECO 312, ORF 245, PHI 201, SOC 404, WWS 200, or WWS 332.",
-      "completed_by_semester": 6,
+      "min_needed": 8,

```

They say "in addition to prerequisites, no fewer than eight", but since the prerequisite courses will inevitably double-count here as well (they're both just `POL *`), the total should be 10 (2 prereqs + 8 other departmentals).

```suggestion
      "min_needed": 10,
```